library/core/src/array/mod.rs

```diff
@@ -344,7 +344,7 @@ impl<T: PartialOrd, const N: usize> PartialOrd for [T; N] {
     }
 }
 
-/// Implements comparison of arrays lexicographically.
+/// Implements comparison of arrays [lexicographically](../std/cmp/trait.Ord.html#lexicographical-comparison).
```

Comment:
This is not an intra-doc link.
```suggestion
/// Implements comparison of arrays [lexicographically](core::cmp::Ord#lexicographical-comparison).
```